### ISSUE TITLE
Need revised CEDS emissions

### ISSUE THREAD

**Issue by rtodling** [GEOS-ESM/GEOSadas#314]
The latest GEOSadas-5.41.2 version of the system does not have the proper emission to allow for its promotion to FPP and FP. 

This needs to be addressed.  Thanks Allie for pointing this out.

**Comment by mathomp4** [comment]
As additional information in re GEOSgcm, the updates needed are detailed in https://github.com/GEOS-ESM/GEOSgcm/pull/803

They are (as far as I know):

1. Update to GOCART `sdr_v2.3.0`
2. Update to a new GEOSchem_GridComp (current `develop`, probably would be tagged v1.15.0)
3. Update to a new GEOSgcm_GridComp (I think all is in `develop` now, either v2.7.0 or maybe in the v2.6 series as the changes needed in GCM GC were for connectivities)

@sdrabenh will discuss with modeling group. These haven't gone in yet only because they are non-zero-diff due to the changed CEDS emissions.

**Comment by rtodling** [comment]
@acollow Allie, does GOCART v2.3.0 not have the updated CEDS emissions? 
@mathomp4 I am looking for sdr_v2.3.0 and that is not in GOCART! 

btw: when I grep for CEDS I see many RC yaml files pointing to that ... so I want to believe that all is good in v2.3.0

**Comment by acollow** [comment]
GOCART v2.3 is the correct version that should be used by GEOSgcm and GEOSadas, and resolved the issue with the CEDS emissions. I want to emphasize that any GOCART tag beginning with sdr should not be used for a system in line for "ops" as those are not official GOCART releases. As long as GEOSadas is now using GOCART v.2.3 this issue can be closed.

**Comment by rtodling** [comment]
@acollow thank you. This will close automatically as we make the next update in this repo.

**Comment by mathomp4** [comment]
Yes. I just compared the old `sdr_v2.3.0` and the "real" `v2.3.0` and `v2.3.0` is the one to use. It's "better" than the sdr_ tag.

**Comment by rtodling** [comment]
> Yes. I just compared the old `sdr_v2.3.0` and the "real" `v2.3.0` and `v2.3.0` is the one to use. It's "better" than the sdr_ tag.

Thanks for double checking @mathomp4
